calliope/core/model.py

```diff
@@ -148,7 +151,10 @@ def _init_from_model_run(self, model_run, debug_data, debug):
                 os.path.dirname(calliope.__file__), "config", "constraints.yaml"
             )
         )
-        self._add_observed_dict("component_config", component_config)
+        component_config_override = self._add_optimisation_config_override(
```

Comment:
Neither seem particularly self-explanatory but I can't think of anything smarter either besides just going more minimal and calling it just `config` 🤷 (though isn't it really data rather than config ;)?

Reply:
agreed offline: `math`